doc/api/prev_api_changes/api_changes_3.4.0/deprecations.rst

```diff
@@ -38,8 +38,8 @@ Subplot-related attributes and methods
 Some ``SubplotBase`` methods and attributes have been deprecated and/or moved
 to `.SubplotSpec`:
 
-- ``get_geometry`` (use `.SubplotBase.get_subplotspec` instead),
-- ``change_geometry`` (use `.SubplotBase.set_subplotspec` instead),
+- ``get_geometry`` (use ``SubplotBase.get_subplotspec`` instead),
+- ``change_geometry`` (use ``SubplotBase.set_subplotspec`` instead),
```

Comment:
The removal message in 3.6 API changes was not updated like here. But should these alternatives really be made code-style instead of linking to the `Axes` methods instead?

Reply:
Fair question. I consider the changelog a historic document that reflects the state at that time. We also don't go back and change other things in changelog, that are not correct anymore. So, code-style is correct.

Reply:
OK, #24236